containers/sarek/environment.yml

```diff
@@ -0,0 +1,24 @@
+# You can use this file to create a conda environment for this pipeline:
+# conda env create -f environment.yml
+name: sarek-core
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+
+dependencies:
+  - bcftools=1.8
+  - bwa=0.7.17
+  - conda-forge::openjdk=8.0.144 # Needed for FastQC docker - see bioconda/bioconda-recipes#5026
+  - fastqc=0.11.7
+  - freebayes=1.2.0
+  - gatk4=4.0.3.0
+  - htslib=1.7
+  - igvtools=2.3.93
+  - manta=1.3.0
```

Comment:
Since we're updating, we can try the 1.4.0

Reply:
OK, done